### DISCUSSION TITLE
Cannot connect Prisma with my MongoDB database

### DISCUSSION THREAD

**Discussion by ArthurGerbelot** [prisma/prisma#9113] [Q&A]
Hi everyone, I'm new to Prisma and I try to run the [Start From Scratch on MongoDB](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/mongodb-typescript-mongodb) project but cannot make it works.

I have a MongoDB running well on version 4.2.12

It's working fine when I'm using mongo cli:
```
mongo mongodb://_______:______@127.0.0.1:27017/devmeteor
MongoDB shell version v3.6.8
connecting to: mongodb://127.0.0.1:27017/devmeteor
Implicit session: session { "id" : UUID("_______________________") }
MongoDB server version: 4.2.12
WARNING: shell and server versions do not match
mongo-rs:PRIMARY>
```

But when I try to set this  on `.env`
`DATABASE_URL="mongodb://____:_____@127.0.0.1:27017/devmeteor"`

And run the project, I have this erreur all the time (even if I try to use it with another Mongodb)
```
npx ts-node src/
(node:10822) UnhandledPromiseRejectionWarning: Error: Database error. error code: unknown, error message: Server selection timeout: No available servers. Topology: { Type: ReplicaSetNoPrimary, Servers: [ { Address: mongo-secondary1:27017, Type: Unknown, Error: failed to lookup address information: Temporary failure in name resolution }, { Address: mongo-master:27017, Type: Unknown, Error: failed to lookup address information: Temporary failure in name resolution }, { Address: preprod-mongo-arbiter:27017, Type: Unknown, Error: failed to lookup address information: Temporary failure in name resolution }, ] }
    at LineStream.<anonymous> (/home/me/test-prisma/node_modules/@prisma/client/runtime/index.js:29704:31)
    at LineStream.emit (events.js:210:5)
    at LineStream.EventEmitter.emit (domain.js:475:20)
    at addChunk (_stream_readable.js:309:12)
    at readableAddChunk (_stream_readable.js:290:11)
    at LineStream.Readable.push (_stream_readable.js:224:10)
    at LineStream.Transform.push (_stream_transform.js:150:32)
    at LineStream._pushBuffer (/home/me/test-prisma/node_modules/@prisma/client/runtime/index.js:26457:19)
    at LineStream._transform (/home/me/test-prisma/node_modules/@prisma/client/runtime/index.js:26451:10)
    at LineStream.Transform._read (_stream_transform.js:189:10)
(node:10822) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:10822) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

I really don't know what to try next to make it works.. Can someone have any idea ? 
Thank all !

**Comment by RichardBray** [comment]
@ArthurGerbelot did you ever find a solution for this issue?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
